lib/charts.rb

```diff
@@ -15,8 +15,8 @@ def initialize(project)
 
     def push(from, to, format)
       @labels << from.strftime(format)
-      @total << project.builds.where("? > builds.created_at AND builds.created_at > ?", to, from).count
-      @success << project.builds.where("? > builds.created_at AND builds.created_at > ?", to, from).success.count
+      @total << project.builds.where("? > builds.created_at AND builds.created_at > ?", to, from).count(:all)
+      @success << project.builds.where("? > builds.created_at AND builds.created_at > ?", to, from).success.count(:all)
```

Comment:
Line is too long. [119/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.